s32/drivers/s32ze/Can_CANEXCEL/CMakeLists.txt

```diff
@@ -4,4 +4,4 @@
 zephyr_include_directories(include)
 zephyr_library_sources(src/CanEXCEL_Ip_HwAccess.c)
 zephyr_library_sources(src/CanEXCEL_Ip_Irq.c)
-zephyr_library_sources(src/CanEXCEL_Ip.c)
+zephyr_library_sources(src/CanEXCEL_Ip.c)
```

Comment:
do not introduce superfluous changes, last blank line must be kept. same applies for any other file.

Reply:
I reverted the last blank line